actions-substream/proto/schema.proto

```
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "sf/ethereum/type/v2/type.proto";
```

Buf (1.55.1)

import "sf/ethereum/type/v2/type.proto": file does not exist

(COMPILE)